docs/dev/git-cheat.md

```diff
@@ -0,0 +1,126 @@
+# Git cheat sheet for (busy) contributors
```

Comment:
Thanks for writing this.

I wonder this should be a standalone website, as the content isn't specific to nerdctl

Reply:
Or maybe this should be merged into https://github.com/containerd/project/blob/main/CONTRIBUTING.md

Reply:
we may add it to [fay](https://github.com/containerd/nerdctl/blob/main/docs/faq.md) too